### PR TITLE
Feature/list

### PR DESCRIPTION
**Pull request by sanjitsarkar**
Added list styling

### CODE REVIEW

**Review by bossmonkey98 — APPROVED** [review]

File: components/css/style.css [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+code {
+  min-width: auto;
+  max-width: fit-content;
+}
+.code-toolbar {
+  min-width: auto;
+  max-width: fit-content;
+}
+button.copy-to-clipboard-button {
+  background-color: var(--light-color) !important;
```
Comment (sanu29) [inline components/css/style.css:135]: try not to use !important

File: components/css/style.css [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  padding: 0.2rem 0.5rem !important;
```
Comment (sanu29) [inline components/css/style.css:136]: use px if when instead of decimal rems

**Review by sanu29 — APPROVED** [review]
Good to merge for now

File: components/css/style.css (hunk 1 above)
Comment (sanjitsarkar) [reply components/css/style.css:135]: We can't change the property of the predefined class of prismjs.
So I override it with !important

**Pull request merged**